src/DayColumn.js

```diff
@@ -160,31 +160,31 @@ class DaySlot extends React.Component {
       availabilityWrapperComponent: AvailabilityWrapper,
       availabilityStartAccessor,
       availabilityEndAccessor,
+      entityKeyAccessor,
       min,
-      date,
     } = this.props;
     let AvailabilityComponent = availabilityComponent;
     let styledAvailabilities = getStyledAvailabilities({
-      availabilities, availabilityStartAccessor, availabilityEndAccessor, min, totalMin: this._totalMin, date
+      availabilities, availabilityStartAccessor, availabilityEndAccessor, min, totalMin: this._totalMin
     });
 
-    return styledAvailabilities.map(({availability, style}, index) => {
+    return styledAvailabilities.map(({availability, style}, idx) => {
       let { height, top } = style;
+      const key = entityKeyAccessor && availability[entityKeyAccessor]
+        ? availability[entityKeyAccessor]
+        : `avbl_${idx}`;
 
-      return AvailabilityWrapper && (
-        <AvailabilityWrapper key={index}>
+      return AvailabilityComponent && (
```

Comment:
Why move the check for `AvailabilityComponent` up to here? We should just be consistent with event rendering, which always renders `EventWrapper` and then tries to render `EventComponent` inside of it if available.